src/WarcraftLegacies.Source/Setup/FactionSetup/StormwindSetup.cs

```diff
@@ -104,7 +104,7 @@ public static void Setup()
         IconName = "Angel",
         Name = "City of Heroes",
         HeroGlowAbilityTypeId = Constants.ABILITY_A0GK_HERO_GLOW_ORIGIN,
-        Filter = unit => !unit.IsType(UNIT_TYPE_MECHANICAL),
+        Filter = unit => !unit.IsType(UNIT_TYPE_MECHANICAL) || unit.GetTypeId() != Constants.UNIT_H05F_STORMWIND_CHAMPION_STORMWIND_ELITE,
```

Comment:
I think this needs to be an AND condition?

Reply:
I did test it in game and it seemed to work. But I can double check.